config/projects.yaml

```diff
@@ -149,14 +149,14 @@ classification-bundle:
             variants:
                 symfony/symfony: ['4.4']
                 sonata-project/admin-bundle: ['3']
-            services: [mongodb]
+            services: [mongodb, pecl]
```

Comment:
WDYT about keeping
```
 tools: {{ ['composer:v' ~ {{ project.composerVersion }}]|merge(branch.tools)|join(, ) }}
```
and renaming the option `services` to `tools` instead ?

And I was expecting we do the smae changes here: https://github.com/sonata-project/dev-kit/blob/master/templates/project/.github/workflows/qa.yaml.twig#L34
and here: https://github.com/sonata-project/dev-kit/blob/master/templates/project/.github/workflows/qa.yaml.twig#L73

Reply:
In project we are using `services`, in `shivammathur/setup-php` config we are using `tools`.
Also services are type `Service[]`, so it will be much work to do for nothing IMO.

@OskarStark WDYT?

Reply:
`Service` could be renamed `Tool`.
The renaming is fast, that's not so much work.
The only usage of this option is here, so it makes sens to call it correctly.

The qa.yaml build should be updated anyway.